programs/programs/ma/cpp/tests/test_cpp.py

```
@@ -0,0 +1,325 @@
+"""
+Unit tests for MaCpp calculator class.
+
+These tests verify the Cambridge Preschool Program (CPP) calculator logic,
+including:
+- Calculator registration and class attributes
+- Cambridge residency eligibility
+- Child age eligibility (ages 3–4)
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)